lib/cocoapods_bugsnag.rb

```diff
@@ -12,7 +12,8 @@ class Installer::UserProjectIntegrator::TargetIntegrator
   # If not present, attempt to lookup the value from the Info.plist
   unless api_key
     default_info_plist_location = Dir.glob("./{ios/,}*/Info.plist").reject {|path| path =~ /build|test/i }
-    plist_buddy_response = `/usr/libexec/PlistBuddy -c "print :BugsnagAPIKey" "#{default_info_plist_location.first}"`
+    plist_buddy_response = `/usr/libexec/PlistBuddy -c "print :bugsnag:apiKey" "#{default_info_plist_location.first}"`
+    plist_buddy_response = `/usr/libexec/PlistBuddy -c "print :BugsnagAPIKey" "#{default_info_plist_location.first}"` if !$?.success?
     api_key = plist_buddy_response if $?.success?
```

Comment:
Maybe it's just me, but I find this quite a lot to take in when it's not doing anything particularly complex.  I'd consider renaming variables and bringing the `.first` up to give something like:
```
    plist = Dir.glob("./{ios/,}*/Info.plist").reject{|path| path =~ /build|test/i }.first
    response = `/usr/libexec/PlistBuddy -c "print :bugsnag:apiKey" "#{plist}"`
    response = `/usr/libexec/PlistBuddy -c "print :BugsnagAPIKey" "#{plist}"` if !$?.success?
    api_key = response if $?.success?
```